apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V80__new_schema_beneficiaries_history.sql

```diff
@@ -0,0 +1,104 @@
+-- target table. PSQL can derive the table structure based on
+-- the data input (i.e., column name, data type). Thus, for HSQL,
+-- we need to explicitly define the table structure prior to loading data.
+--
+-- For HSQL, explicitly define/create a new BENEFICIARIES_HISTORY_NEW table in
+-- the current PUBLIC schema
+--
+${logic.hsql-only}  create table public.beneficiaries_history_new (
+${logic.hsql-only}    bene_history_id      bigint not null,
+${logic.hsql-only}    bene_id              bigint not null,
+${logic.hsql-only}    hicn_unhashed        varchar(11) not null,
+${logic.hsql-only}    mbi_num              varchar(11),
+${logic.hsql-only}    efctv_bgn_dt         date,
+${logic.hsql-only}    efctv_end_dt         date,
+${logic.hsql-only}    bene_crnt_hic_num    varchar(64) not null,
+${logic.hsql-only}    mbi_hash             varchar(64) not null,
+${logic.hsql-only}    bene_birth_dt        date not null,
+${logic.hsql-only}    bene_sex_ident_cd    char(1) not null,
+${logic.hsql-only}    last_updated         timestamp with time zone,
+${logic.hsql-only} constraint public.beneficiaries_history_new_pkey
+${logic.hsql-only} primary key (bene_history_id) );
+--
+--
+${logic.psql-only} SET max_parallel_workers = 24;
+${logic.psql-only} SET max_parallel_workers_per_gather = 20;
+${logic.psql-only} SET parallel_leader_participation = off;
+${logic.psql-only} SET parallel_tuple_cost = 0;
+${logic.psql-only} SET parallel_setup_cost = 0;
+--
+--
+${logic.hsql-only} insert into public.beneficiaries_history_new (
+${logic.hsql-only}    bene_history_id,
+${logic.hsql-only}    bene_id,
+${logic.hsql-only}    hicn_unhashed,
+${logic.hsql-only}    mbi_num,
+${logic.hsql-only}    efctv_bgn_dt,
+${logic.hsql-only}    efctv_end_dt,
+${logic.hsql-only}    bene_crnt_hic_num,
+${logic.hsql-only}    mbi_hash,
+${logic.hsql-only}    bene_birth_dt,
+${logic.hsql-only}    bene_sex_ident_cd,
+${logic.hsql-only}    last_updated )
+--
+-- PSQL allows us to dynamically create a table via the
+-- associated SELECT statement used to populate the table.
+--
+${logic.psql-only} create table public.beneficiaries_history_new as
+select 
+${logic.hsql-only}  convert(bene_history_id, SQL_BIGINT),
+${logic.hsql-only}  convert(bene_id, SQL_BIGINT),
+${logic.psql-only}  bene_history_id::bigint,
+${logic.psql-only}  bene_id::bigint,
+                    hicn_unhashed,
+                    mbi_num,
+                    efctv_bgn_dt,
+                    efctv_end_dt,
+                    bene_crnt_hic_num,
+                    mbi_hash,
+                    bene_birth_dt,
+                    bene_sex_ident_cd,
+                    last_updated
+from
+    public.beneficiaries_history;
+    
+${logic.psql-only} alter table public.beneficiaries_history_new
+${logic.psql-only}    alter column bene_history_id     SET NOT NULL,
+${logic.psql-only}    alter column bene_id             SET NOT NULL,
+${logic.psql-only}    alter column hicn_unhashed       SET NOT NULL,
```

Comment:
Looks like this is currently nullable in prod

Reply:
fixed.